<launch>
	<arg name="init" default="false"/>
	<node name="calculator" pkg="beginner_tutorial" type="kuroki" output="screen" launch-prefix="gnome-terminal --command" />
	<node name="view_point" pkg="beginner_tutorial" type="view_point_x_y_theta_array" output="screen" launch-prefix="gnome-terminal --command" />
	<node name="target_position" pkg="beginner_tutorial" type="target_x_y_array" output="screen" launch-prefix="gnome-terminal --command" />
	<node name="horizontal_diviation" pkg="beginner_tutorial" type="horizontal_diviation" output="screen" launch-prefix="gnome-terminal --command" />
	<node name="steering" pkg="beginner_tutorial" type="steering" output="screen" launch-prefix="gnome-terminal --command" />
	<node name="plotter" pkg="beginner_tutorial" type="plot.py" output="screen" launch-prefix="gnome-terminal --command" />
	<node if="$(arg init)" name="first_publisher" pkg="beginner_tutorial" type="first_publisher" output="screen" launch-prefix="gnome-terminal --command" />
</launch>
